vm/ubpf_vm.c

```diff
@@ -144,7 +144,7 @@ u32(uint64_t x)
 }
 
 int 
-ubpf_exec(const struct ubpf_vm *vm, void *mem, size_t mem_len, uint64_t* bpf_return_value)
+ubpf_exec(struct ubpf_vm *vm, void *mem, size_t mem_len, uint64_t* bpf_return_value)
```

Comment:
This patch is a bit of a mess: contains part of the new interface, changes to the library and changes to the test executable. Please at least separate library changes and test executable changes.

Reply:
Seperated in next version @lsgunth